sources/api/schnauzer/src/helpers.rs

```diff
@@ -239,6 +239,9 @@ mod error {
         #[snafu(display("Unknown architecture '{}' given to goarch helper", given))]
         UnknownArch { given: String },
 
+        #[snafu(display("Invalid Cidr format '{}'", cidr))]
+        InvalidCidr { cidr: String },
+
```

Comment:
It seems like we also throw this on IP address parse errors. I'd suggest splitting it into two:

```suggestion
        #[snafu(display("Invalid Cidr format '{}'", cidr))]
        InvalidCidr { cidr: String },
        
        #[snafu(display("Invalid IP Address format '{}': {}", cidr, source))]
        InvalidIPAddress { ip_address: String, source: std::net::AddrParseError },

```